tools/cpp/windows_cc_configure.bzl

```diff
@@ -190,15 +190,17 @@ def _find_vcvarsall_bat_script(repository_ctx, vc_path):
   return vcvarsall
 
 
-def _find_env_vars(repository_ctx, vc_path):
+def setup_vc_env_vars(repository_ctx, vc_path):
```

Comment:
Excuse me, @meteorcloudy, but how would we access `repository_ctx` and `vc_path`?
Ideally, I would like to make macro like:

```
def generate_res(rc):
  native.genrule(name="generate-res", srcs=[rc], outs=[rc+".res"], cmd="$(RC) /fo$@ $<")
```

Reply:
They are only access in repository rules:
https://docs.bazel.build/versions/master/skylark/repository_rules.html